### PR TITLE
Remove attached property. Android bug fix. Added ability to set ErrorImageTintColor.

### PR DESCRIPTION
**Pull request by stackunderflows**

### CODE REVIEW

File: Library/PhantomLib.Droid/Renderers/UltimateEntryRenderer.cs [hunk 1]
```diff
@@ -118,6 +118,20 @@ void EditText_FocusChange(object sender, FocusChangeEventArgs e)
...
+            {
+#pragma warning disable CS0618 // Type or member is obsolete
+                drawable.SetColorFilter(color.ToAndroid(), PorterDuff.Mode.SrcIn);
+#pragma warning restore CS0618 // Type or member is obsolete
+            }
+        }
+
         public void UpdateControlUI()
         {
             GradientDrawable gradientDrawable = new GradientDrawable();
```
Comment (willmetz) [inline Library/PhantomLib.Droid/Renderers/UltimateEntryRenderer.cs]: is this gradientDrawable needed?

File: Library/PhantomLib.Droid/PhantomLib.Droid.csproj [hunk 2]
```diff
@@ -58,9 +58,6 @@
   <ItemGroup>
     <None Include="Resources\AboutResources.txt" />
   </ItemGroup>
-  <ItemGroup>
```
Comment (willmetz) [inline Library/PhantomLib.Droid/PhantomLib.Droid.csproj:61]: just wanted to check that removing this was intentional

File: Library/PhantomLib.Droid/Renderers/UltimateEntryRenderer.cs (hunk 1 above)
Comment (stackunderflows) [reply Library/PhantomLib.Droid/Renderers/UltimateEntryRenderer.cs]: Nope. I removed it.

File: Library/PhantomLib.Droid/PhantomLib.Droid.csproj (hunk 2 above)
Comment (stackunderflows) [reply Library/PhantomLib.Droid/PhantomLib.Droid.csproj:61]: I had to add Xamarin.Forms to this project, which pulled in a metric crap ton of Android support packages with it.

**Review by willmetz — APPROVED** [review]

**Pull request merged**